util/pkg/vfs/s3fs.go

```diff
@@ -61,6 +61,14 @@ func newS3Path(s3Context *S3Context, bucket string, key string) *S3Path {
 }
 
 func (p *S3Path) Path() string {
+	response,_ := p.GetVersioning()
+	if response != true {
+		fmt.Println("S3 Bucket Versioning is not set. Let's set it")
+		p.SetVersioning("Enabled")
+	} else {
+		fmt.Println("S3 Bucket Versioning is set already.")
```

Comment:
Just pull the else, unless you want to put in a glog that is like V(8).

Reply:
done